<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckUpdatesDialog</class>
 <widget class="QDialog" name="CheckUpdatesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Updates</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="newVersion_label">
     <property name="text">
      <string>Available FMIDownloader version: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="currentVersion_label">
     <property name="text">
      <string>Your current version is: </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkUpdatesOnStartUp_Checkbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Check updates on startup</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="goToDownloads_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Go to downloads</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CheckUpdatesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
